hq/public/stylesheets/main.css

```diff
@@ -423,3 +397,27 @@ table.gcp-report__table {
   height: 40px;
 }
 
+/* GCP */
+table.gcp-report__table td,
+table.gcp-report__table th {
+  min-height: 21px;
+  box-sizing: initial;
+  word-wrap: break-word;
+}
+
+table.gcp-report__table {
```

Comment:
I think you don't need to bother with the `table.` selector as you already have the class name - think we discussed this on a previous PR (I might be wrong though!)